### PR TITLE
[Speech] Update bindings for Xcode 12.5 Beta 1

### PR DESCRIPTION
**Pull request by rachelkang**

### CODE REVIEW

File: src/speech.cs [hunk 1]
```diff
@@ -262,6 +290,7 @@ interface SFTranscriptionSegment : NSCopying, NSSecureCoding {
 
 		[iOS (13, 0)]
 		[NullAllowed, Export ("voiceAnalytics")]
+		[Advice ("Use 'SFVoiceAnalytics' from 'SFSpeechRecognitionMetadata' instead.")]
```
Comment (dalexsoto) [inline src/speech.cs:293]: Small typo here

```suggestion
		[Advice ("Use 'VoiceAnalytics' from 'SFSpeechRecognitionMetadata' instead.")]
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]

**Review by tj-devel709 — APPROVED** [review]
Congrats on first PR! Good for me after Alex's suggestion! :)

**Review by spouliot — APPROVED** [review]

**Comment by spouliot** [comment]
removed `note-deprecation` since we do not use it inside release notes for API (there's too much of them in general)

File: src/speech.cs (hunk 1 above)
Comment (rachelkang) [reply src/speech.cs:293]: oops! thanks for catching

**Review by dalexsoto — APPROVED** [review]
👍

**Pull request merged**